scripts/spell_check/spelling.dat

```diff
@@ -2390,6 +2390,8 @@ dosent':doesn't
 dosen't:doesn't
 doub:doubt
 doulbe:double
+double click:double-click
+doubleclick:double-click
```

Comment:
you guess right :'( and I will run check_spelling here again. Thanks for the advise.